packages/client/src/utils/Env.re

```diff
@@ -1,17 +1,22 @@
 [@bs.val] external apiUrlEnv: string = "window.ENV.API_URL";
 [@bs.val] external auth0DomainEnv: string = "window.ENV.AUTH0_DOMAIN";
 [@bs.val] external auth0ClientIdEnv: string = "window.ENV.AUTH0_CLIENT_ID";
+[@bs.val] external modeEnv: string = "window.ENV.MODE";
 [@bs.val] external origin: string = "window.location.origin";
 
 let auth0Domain = auth0DomainEnv;
 let auth0ClientId = auth0ClientIdEnv;
 let redirectUrl = origin ++ "/callback";
 let logoutUrl = origin ++ "/login";
 let serverUrl = apiUrlEnv;
+let mode = modeEnv;
```

Comment:
what is "MODE"? Maybe "CLIENT_ENV" or "JS_ENV" or something?

Reply:
Think that "CLIENT_ENV" is better.